conf/machine/imx8qxpc0mek.conf

```diff
@@ -0,0 +1,8 @@
+#@TYPE: Machine
+#@NAME: i.MX 8QXP MEK
+#@DESCRIPTION: i.MX 8QXP C0 Multisensory Enablement Kit (MEK) board
+#@MAINTAINER: Lauren Post <lauren.post@nxp.com>
+
+MACHINEOVERRIDES =. "mx8qxpc0:"
```

Comment:
What is different?

Reply:
`MACHINEOVERRIDES`.

Reply:
Sure but how it inpact the system and why this is not detected on runtime?

Reply:
The SECO FW is different for B0 and C0 and cannot be chosen at runtime. Apparently it would defeat the secure purpose of the firmware.